textstat/textstat.py

```diff
@@ -26,6 +26,13 @@ def legacy_round(number, points=0):
     p = 10 ** points
     return float(math.floor((number * p) + math.copysign(0.5, number))) / p
 
+ordinal_map = {1: 'th', 2: 'nd', 3: 'rd'}
+teens_map = {11: 'th', 12: 'th', 13: 'th'}
```

Comment:
Could we drop `ordinal_map` and `teens_map` inside the `get_grade_suffix` function? I don't think they need to be accessible to anything but that function.